src/init/starship.ps1

```diff
@@ -5,6 +5,8 @@
 function global:prompt {
     $origDollarQuestion = $global:?
     $origLastExitCode = $global:LASTEXITCODE
+    $origCaretValue = ${global:^}
+    $origLastErrorCmdletName = Get-Error |  Where-Object {$_ -ne $null} | Select-Object -expand InvocationInfo | Where-Object {$_.InvocationName -ne $null} | Select-Object -expand InvocationName
```

Comment:
I ist possible to initialize this when it's needed?

Reply:
Done, also check previous comment for clarification about moving this login to $lastCmd if block.